doc/classes/TextEdit.xml

```diff
@@ -205,10 +241,9 @@
 				Returns the caret pixel draw position.
 			</description>
 		</method>
-		<method name="get_caret_index_edit_order">
+		<method name="get_caret_index_edit_order" deprecated="Carets no longer need to be edited in any specific order. If the carets need to be sorted, use [method get_sorted_carets] instead.">
 			<return type="PackedInt32Array" />
 			<description>
-				Returns a list of caret indexes in their edit order, this done from bottom to top. Edit order refers to the way actions such as [method insert_text_at_caret] are applied.
 			</description>
```

Comment:
`get_caret_index_edit_order` does still work, but the description is outdated since there is no more required edit order of carets.
Should I keep the description as it was or update it somehow?